github/enterprise_app_installation.go

```diff
@@ -0,0 +1,158 @@
+	ClientID string `json:"client_id"`
+	// The selection of repositories that the GitHub app can access.
+	// Can be one of: all, selected, none
+	RepositorySelection string `json:"repository_selection"`
+	// A list of repository names that the GitHub App can access, if the repository_selection is set to selected.
+	Repository []string `json:"repository,omitempty"`
+}
+
+// ListInstallableEnterpriseOrgs lists the organizations in an enterprise that are installable for an app.
+//
+// GitHub API docs: https://docs.github.com/enterprise-cloud@latest/rest/enterprise-admin/organization-installations#get-enterprise-owned-organizations-that-can-have-github-apps-installed
+//
+//meta:operation GET /enterprises/{enterprise}/apps/installable_organizations
+func (s *EnterpriseService) ListInstallableEnterpriseOrgs(ctx context.Context, enterprise string, opts *ListOptions) ([]*InstallableOrganization, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/apps/installable_organizations", enterprise)
+
+	u, err := addOptions(u, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var orgs []*InstallableOrganization
+	resp, err := s.client.Do(ctx, req, &orgs)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return orgs, resp, nil
+}
+
+// ListEnterpriseOrgAccessibleRepositories lists the repositories accessible to an app in an enterprise-owned organization.
+//
+// GitHub API docs: https://docs.github.com/enterprise-cloud@latest/rest/enterprise-admin/organization-installations#get-repositories-belonging-to-an-enterprise-owned-organization
+//
+//meta:operation GET /enterprises/{enterprise}/apps/installable_organizations/{org}/accessible_repositories
+func (s *EnterpriseService) ListEnterpriseOrgAccessibleRepositories(ctx context.Context, enterprise, org string, opts *ListOptions) ([]*AccessibleRepository, *Response, error) {
+	u := fmt.Sprintf("enterprises/%v/apps/installable_organizations/%v/accessible_repositories", enterprise, org)
+
+	u, err := addOptions(u, opts)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var repos []*AccessibleRepository
+	resp, err := s.client.Do(ctx, req, &repos)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return repos, resp, nil
+}
+
+// ListEnterpriseAppInstallationForOrg lists the GitHub app installations associated with the given enterprise-owned organization.
+//
+// GitHub API docs: https://docs.github.com/enterprise-cloud@latest/rest/enterprise-admin/organization-installations#list-github-apps-installed-on-an-enterprise-owned-organization
+//
+//meta:operation GET /enterprises/{enterprise}/apps/organizations/{org}/installations
+func (s *EnterpriseService) ListEnterpriseAppInstallationForOrg(ctx context.Context, enterprise, org string, opts *ListOptions) ([]*Installation, *Response, error) {
```

Comment:
See https://github.com/google/go-github/issues/3761

```suggestion
func (s *EnterpriseService) ListAppOrganizationInstallations(ctx context.Context, enterprise, org string, opts *ListOptions) ([]*Installation, *Response, error) {
```